src/checkers/inference/model/ArithmeticConstraint.java

```diff
@@ -57,6 +57,16 @@ public static ArithmeticOperationKind fromTreeKind(Kind kind) {
                     return OR;
                 case XOR:
                     return XOR;
+                case PLUS_ASSIGNMENT:
+                    return PLUS;
```

Comment:
That the two versions are handled the same would be clearer if you used fall-through cases:

````
case PLUS:
case PLUS_ASSIGNMENT:
    return PLUS;
````